speech/system-test/quickstart.test.js

```diff
@@ -15,11 +15,17 @@
 
 'use strict';
 
-require(`../../system-test/_setup`);
-
 const path = require(`path`);
 const proxyquire = require(`proxyquire`).noPreserveCache();
+const sinon = require(`sinon`);
 const speech = proxyquire(`@google-cloud/speech`, {})();
+const test = require(`ava`);
+
+const {
```

Comment:
I don't think Node v4 supports this format.

(Caveat: I think we may have talked about only supporting Node v6 in our tests, but I can't recall. If that's what we do, there's a [small] risk our samples may pass on Node v6 but fail on Node v4.)